NineChronicles.Headless/GraphQLService.cs

```diff
@@ -190,6 +205,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseMiddleware<HttpCaptureMiddleware>();
 
                 app.UseMiddleware<LocalAuthenticationMiddleware>();
+                app.UseMiddleware<JwtAuthenticationMiddleware>();
```

Comment:
```suggestion
                if (Convert.ToBoolean(Configuration.GetSection("Jwt")["EnableJwtAuthentication"]))
                {
                    app.UseMiddleware<JwtAuthenticationMiddleware>();
                }
```
Don't run if `Jwt` is disabled.